src/python/zquantum/vqe/singlet_uccsd.py

```diff
@@ -144,9 +136,9 @@ def screen_out_operator_terms_below_threshold(
         amplitudes = np.array(amplitudes)
         return amplitudes, new_fermion_generator
 
-    def generate_circuit_from_mp2_amplitudes(
+    def _compute_uccsd_vector_from_mp2_amplitudes(
         self, raw_mp2_operator: FermionOperator, screening_threshold: float = 0.0
```

Comment:
Same comment about not explicitly using `mp2`. Could rename `raw_mp2_operator` to something like `fermion_generator`.